python/lsst/ts/genericcamera/genericcameracsc.py

```diff
@@ -190,6 +206,21 @@ async def end_disable(self, id_data):
             finally:
                 self.server = None
 
+        if self.s3bucket is not None:
+            self.s3bucket.stop_mock()
+        self.s3bucket = None
+        self.log.info("end_disable")
+
+    async def begin_standby(self, id_data):
```

Comment:
`id_data` -> `data`

I noticed that other methods are still using the old design `id_data`, but since you are adding this method now, maybe consider using the more recent name convention. Would be good to update all `id_data` to `data`, but up to you if you want to do it now or on a separate ticket. I would probably do it in a separate ticket is it was me.

Reply:
Done.

Reply:
Instead of doing a text reply, try the 👍 emoji response.. That generates less e-mail traffic... 😄 

https://developer.lsst.io/work/flow.html#code-review-discussion